### ISSUE TITLE
Feature Request: Automatic issue generation and score threshold to ignore for automatic issue generation

### ISSUE THREAD

**Issue by billdenney** [DylanDijk/CTVsuggest#3]
I think that it would be useful to regularly (something like monthly or quarterly) run the package on all CTVs, and automatically generate GitHub issues for them.  (I'd guess that this could be done via GitHub continuous integration, but I'm not sure.)

For that, I think it would help to filter only packages that have a score greater than some threshold.  Similar to #2, that threshold could be given in the yaml header with a key like `CTVsuggest_auto_issue_threshold: 0.5`.  For CTV that have that flag set, they get the automatic issue, only if there are packages that have a score greater than the threshold.  If the flag is not set, then that is a way that the maintainer could automatically opt out.

Alternatively, you could have an explicit opt out by setting `CTVsuggest_auto_issue_threshold: 1` (that may be better since default opting out could cause things to be missed).

**Comment by DylanDijk** [comment]
I agree, this is another good idea. I have opened https://github.com/cran-task-views/ctv/issues/51 on the CTV repository, as I wanted to check first if the CTV maintainers are happy to have these changes to the Task View source files. Do you think this necessary or should we just go ahead and implement this?

**Comment by billdenney** [comment]
It seems like a good thing to ask.  It would affect all CTV maintainers, so they should weigh in.

**Comment by DylanDijk** [comment]
Ok, we will first see what the conclusion is there.
